event/remote.go

```diff
@@ -46,6 +46,18 @@ const (
 	// (logbook + versions) remove completed
 	// payload will be a RemoteEvent
 	ETRemoteClientRemoveDatasetCompleted = Type("remoteClient:RemoveDatasetCompleted")
+	// ETRemoteDatasetFollowed indicates that the dataset has been followed by a user
+	// payload is an `InitID` string
+	ETRemoteDatasetFollowed = Type("remote:DatasetFollowed")
+	// ETRemoteDatasetUnfollowed indicates that the dataset has been unfollowed by a user
+	// payload is an `InitID` string
+	ETRemoteDatasetUnfollowed = Type("remote:DatasetUnfollowed")
+	// ETRemoteDatasetIssueOpened indicates that an issue has been opened for this dataset
+	// payload is an `initID` string
+	ETRemoteDatasetIssueOpened = Type("remote:DatasetIssueOpened")
+	// ETRemoteDatasetIssueClosed indicates that an issue has been closed for this dataset
+	// payload is an `initID` string
+	ETRemoteDatasetIssueClosed = Type("remote:DatasetIssueClosed")
```

Comment:
@Arqu took these from issue #1880, but wanted to get a double check on if these work for cloud, if you think we should rename, etc!

Reply:
These look great, cloud is a bit out of date on the formatting/naming so have some clean up to do. Made an issue to keep it on my plate.

Reply:
got the verbal okay from @Arqu that these looked good